src/Pages/CropInfoPage.js

```diff
@@ -2,10 +2,12 @@ import { FormGroup, Snackbar, Typography } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import { Alert } from '@material-ui/lab';
 import CropForm from 'components/common/CropForm';
+import DeleteForm from 'components/common/DeleteForm';
 import SimpleModal from 'components/common/SimpleModal';
 import TableComponent from 'components/common/TableComponent';
 import { getAllCrops } from 'dataclients/CropsClient';
 import React, { useEffect, useState } from 'react';
+import { deleteCropById } from 'dataclients/CropsClient';
```

Comment:
Can we just call the API deleteCrop, deleteField etc , the way we do for update and create scenarios?